lockup/LockupLinearCurvesCreator.sol

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.22;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { ud60x18 } from "@prb/math/src/UD60x18.sol";
+import { ISablierLockup } from "@sablier/lockup/src/interfaces/ISablierLockup.sol";
+import { Broker, Lockup, LockupLinear } from "@sablier/lockup/src/types/DataTypes.sol";
+
+/// @notice Examples of how to create Lockup Linear streams with different curve shapes.
+/// @dev A visualization of the curve shapes can be found in the docs:
+/// https://docs.sablier.com/concepts/lockup/stream-shapeslockup-linear
+/// Visualizing the curves while reviewing this code is recommended. The X axis will be assumed to represent "days".
+contract LockupLinearCurvesCreator {
+    // Sepolia addresses
+    IERC20 public constant DAI = IERC20(0x68194a729C2450ad26072b3D33ADaCbcef39D574);
+    ISablierLockup public constant LOCKUP = ISablierLockup(0xC2Da366fD67423b500cDF4712BdB41d0995b0794);
+
+    /// @dev For this function to work, the sender must have approved this dummy contract to spend DAI.
+    function createStream_Linear() public returns (uint256 streamId) {
```

Comment:
Should we use a more explainable name similar to Dynamic, such as `createStream_LinearWithoutCliff`?